parse/EffectParser4.cpp

```diff
@@ -87,9 +89,10 @@ namespace {
                 ;
 
             create_field_1
-                =   (       tok.CreateField_
-                        >>  parse::detail::label(Type_token)    >>  parse::string_value_ref() [ _a = _1 ]
-                        >>  parse::detail::label(Size_token)    >   parse::double_value_ref() [ _b = _1 ]
+                =   ((       tok.CreateField_
+                     >>  parse::detail::label(Type_token)    >>  parse::string_value_ref() [ _a = _1 ]
+                     >>  parse::detail::label(Size_token)
+                     ) >   parse::double_value_ref() [ _b = _1 ]
```

Comment:
in cases like this, put the `>` on the next line and align the `parse` with the previous line